src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/Container/GetDestinationSharesForContainerMigration.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Management.Automation;
+using Microsoft.AzureStack.AzureConsistentStorage;
+using Microsoft.AzureStack.AzureConsistentStorage.Models;
+
+namespace Microsoft.AzureStack.AzureConsistentStorage.Commands
+{
+    /// <summary>
+    /// SYNTAX
+    /// Get-ACSDestinationSharesForContainer  [-SubscriptionId] {string} [-Token] {string} [-AdminUri] {Uri} [-ResourceGroupName] {string} 
+    ///                  [-FarmName] {string} [-SourceShareName] {string} [{CommonParameters}] 
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, Nouns.AdminDestinationShareForContainer)]
+    public sealed class GetDesinationSharesForContainerMigration : AdminCmdlet
+    {
+        /// <summary>
+        /// Resource group name
+        /// </summary>
+        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, Position = 3)]
```

Comment:
remove position everywhere or assign the correct value. I think AZ PS doesn't recommend more than one or two positional parameters